lbr.py

```diff
@@ -691,11 +699,14 @@ def nc(x): return 'non-cold ' + x
                    prefix='misprediction-ratio', comment='paths histogram')
 
 def print_common(total):
+  def print_notes():
+    print('# CMP denotes cmp or test instructions\n')
   if glob['size_stats_en']:
     totalv = (total - stat['bad'] - stat['bogus'])
     stat['size']['avg'] = round(stat['size']['sum'] / totalv, 1) if totalv else -1
   print('LBR samples:', hist_fmt(stat))
   if edge_en and total: print_global_stats()
+  print_notes()
   print('#Global-stats-end\n')
```

Comment:
How about denser one call of:
print('\n'.join(["# CMP ..", "#Global-stats-end", ""])

Reply:
I used this so we can add notes easily in future commits to print_notes()